tests/test_io.py

```diff
@@ -149,3 +155,24 @@ def test_read_color_off():
 
     assert_points_xyz(color_off)
     assert_points_color(color_off)
+
+
+def test_read_bin():
+    arr = PyntCloud.from_file(data_path + '.bin')
+
+    assert_points_xyz(arr)
+
+
+def test_write_bin():
```

Comment:
I know that the existing test suite for the io module is not very complete but would you mind to add an extra test to assert that your custom exception is raised??

Maybe something similar to https://github.com/daavoo/pyntcloud/blob/e9df058e7a8078bacea8507d35a57063744ac374/tests/integration/samplers/test_points_samplers.py#L37

Reply:
Will add the extra test before I push :+1: